packages/rlc-common/package.json

```diff
@@ -41,7 +41,21 @@
     "@types/lodash": "^4.14.182",
     "fs-extra": "^10.0.0",
     "@types/fs-extra": "^8.1.0",
-    "ts-node": "^10.7.0"
+    "ts-node": "^10.7.0",
+    "mocha": "^10.2.0",
+    "chai": "^4.3.7",
+    "@types/chai": "^4.3.4",
+    "@types/mocha": "^10.0.1"
+  },
+  "mocha": {
+    "extension": [
+      "ts",
+      "tsx"
+    ],
+    "loader": "ts-node/esm",
+    "spec": [
+      "test/**/*.spec.ts"
+    ]
```

Comment:
Is this the recommend way ? I feel like we should be consistent.

Reply:
Update to align with typespec-ts